src/agat/agat_sp_filter_feature_from_kill_list/config.vsh.yaml

```
@@ -0,0 +1,105 @@
+name: agat_sp_filter_feature_from_kill_list
+namespace: agat
+description: |
+  The script aims to remove features based on a kill list. The default
+  behaviour is to look at the features's ID. If the feature has an ID
+  (case insensitive) listed among the kill list it will be removed. /!\
+  Removing a level1 or level2 feature will automatically remove all linked
+  subfeatures, and removing all children of a feature will automatically
+  remove this feature too.
+keywords: [gene annotations]
+links:
+  homepage: https://github.com/NBISweden/AGAT
+  documentation: https://agat.readthedocs.io/en/latest/tools/agat_sp_filter_feature_from_kill_list.html
+  issue_tracker: https://github.com/NBISweden/AGAT/issues
+  repository: https://github.com/NBISweden/AGAT
+references: 
+  doi: 10.5281/zenodo.3552717
+license: GPL-3.0
+authors:
+  - __merge__: /src/_authors/leila_paquay.yaml
+    roles: [ author, maintainer ]
+
+argument_groups:
+  - name: Inputs
+    arguments:
+      - name: --gff
+        alternatives: [-f, --ref, -reffile]
+        description: Input GFF3 file that will be read
+        type: file
+        required: true
+        direction: input
+      - name: --kill_list
+        alternatives: [--kl]
+        description: Text file containing the kill list. One value per line.
+        type: file
+        required: true
+        example: kill_list.txt
+        direction: input
+  - name: Outputs
+    arguments:
+      - name: --output
+        alternatives: [-o, --out]
+        description: |
+          Output GFF file. If no output file is specified, the output will
+          be written to STDOUT.
```

Viash does not work with STDOUT
```suggestion
          Path to the output GFF file that contains filtered features. 
```